rero_ils/modules/mef_persons/templates/rero_ils/detailed_view_persons.html

```diff
@@ -51,6 +51,22 @@ <h1 class="mb-0">{{ record | person_label(current_i18n.language) }}</h1>
     </section>
   </article>
 </article>
+<!-- Documents -->
+{%- if record.documents %}
+<section class="py-4 border-top">
+  <header class="mb-4">
```

Comment:
I think it's not necessary to add mb-4. The h3 below has already a margin-bottom.

Reply:
Took this one from Items on document view.